### ISSUE TITLE
Make Minishift cache global shared between all profiles

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#1408]
Profile needs a shared cache solution. It is too slow to have to redownload everything every time

```
9.20 MiB / 33.92 MiB [============>--------------------------------------------]  27.11% 23m19s
```

**Comment by gbraad** [comment]
With the `--iso-url` option, you can bypass the download, by specifying an ISO from a cache which is already downloaded. If #1409 is available, this can also work for the `oc` client.

**Comment by hferentschik** [comment]
> So, we are using ~/.minishift/cache for the global cache right?

most likely

**Comment by gbraad** [comment]
> most likely

A little late to ask, but it concerns me. Has this been decided/can it be answered ? It is a too vague and should be given before we can start on an implementation. My 2c. I can assume it would have to be... but...

**Comment by gbraad** [comment]
Fixed with 07c8570

**Comment by LalatenduMohanty** [comment]
PR  reference https://github.com/minishift/minishift/pull/1716
